store/search/ts.go

```diff
@@ -210,14 +214,23 @@ func (s *storeImpl) Search(_ context.Context, table string, query *qsearch.Query
 		})
 	}
 
-	res, err := s.client.MultiSearch.Perform(&tsApi.MultiSearchParams{}, tsApi.MultiSearchSearchesParameter{
+	res, err := s.client.MultiSearch.PerformWithContentType(&tsApi.MultiSearchParams{}, tsApi.MultiSearchSearchesParameter{
 		Searches: params,
-	})
+	}, StreamContentType)
 	if err != nil {
 		return nil, s.convertToInternalError(err)
 	}
 
-	return res.Results, nil
+	reader := bytes.NewReader(res.Body)
+	decoder := jsoniter.NewDecoder(reader)
+	decoder.UseNumber()
+
+	var dest tsApi.MultiSearchResult
```

Comment:
For now, we can use it but later we should just unmarshal it into structs defined in the service layer so we can optimize this path and only maintain a single copy of all these internal structs.

Reply:
Yeah makes sense